update-reviewers.py

```diff
@@ -0,0 +1,153 @@
+        try:
+            if pkgName.lower() not in [ac.lower() for ac in existing_contribs]:
+                contribs[gh_user][contrib_type_yml].append(pkgName)
+        except (AttributeError, TypeError) as AE:
+            contribs[gh_user][contrib_type_yml] = [pkgName]
+        # Update their contributor type role - this populates the website
+        if not contribs[gh_user]["contributor_type"]:
+            contribs[gh_user]["contributor_type"] = [review_role]
+        elif review_role not in contribs[gh_user]["contributor_type"]:
+            # Adding vs append() because the review_role can be a list
+            try:
+                contribs[gh_user]["contributor_type"] + review_role
+            except TypeError as te:
+                contribs[gh_user]["contributor_type"].append(review_role)
+
+        # Finally check that peer-review is in the list of types
+        if "peer-review" not in contribs[gh_user]["contributor_type"]:
+            contribs[gh_user]["contributor_type"].append("peer-review")
+
+    # Now parse through all_current_maintainers
+    # Note that the first maintainer may be the submitting author
+    # TODO - is there a way to combine this with the above workflow
+    # the difference is there is a list of maintainers
+    print("Now on to maintainers")
+    maintainer = {"all_current_maintainers": "packages-submitted"}
+    aType = "all_current_maintainers"
+    # If the review has all_current maintainers, use it
+    if aType in issue_meta:
+        for i, anItem in enumerate(issue_meta[aType]):
+            gh_user = anItem["github_username"].lower().strip()
+            if anItem["name"] == "":
+                try:
+                    # Try to populate name from contributors file
+                    packages[pkgName]["all_current_maintainers"][i]["name"] = contribs[
+                        gh_user
+                    ]["name"]
+                    # Add them to the correct maintainer key packages-submitted
+
+                    # If the  list is empty return AE and start a list
+                    if not contribs[gh_user]["packages-submitted"]:
+                        contribs[gh_user]["packages-submitted"] = pkgName
+                    elif pkgName not in contribs[gh_user]["packages-submitted"]:
+                        contribs[gh_user]["packages-submitted"].append(pkgName)
+
+                except KeyError as ke:
+                    #  If there is a keyerror - add them to the contribs dict
+                    print(
+                        "Missing a maintainer/user -",
+                        gh_user,
+                        "No prob! I'll add them later",
+                    )
+                    missing_contribs.append(gh_user)
+    else:
+        print("All maintainers is missing in the review for ", pkgName)
+
+# Export to yaml
+process_contribs.clean_export_yml(contribs, "contribs.yml")
+process_contribs.clean_export_yml(packages, "packs.yml")
```

Comment:
there are issues with the YAML exporting with characters *id001 and &id001 i'm starting to wonder if that is ascii now or something?

Reply:
this has been fixed locally